### DISCUSSION TITLE
How to show formatted json data for response?

### DISCUSSION THREAD

**Discussion by hejiamin** [scalar/scalar#3950] [Q&A] [answered]
I use asp.net core 9 webapi template and Scalar.AspNetCore 1.2.40. 
I want show formatted json data like image 1(use scalar windows desktop app) in browser. but i got original text data(like image 2) actually in browser.

![图片](https://github.com/user-attachments/assets/6be59da9-21b5-4e9d-b831-8913cf1f06a0)

![图片](https://github.com/user-attachments/assets/abf2ee3d-4990-4a28-80b7-46a276ff530c)

**Comment by hanspagel** [comment, marked as answer]
Fixed! :)

**Reply by hejiamin** [reply]
thanks!
